client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -47,15 +47,16 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
           <TextField
+            className={`${!(touched.email && Boolean(errors.email)) && classes.errorOffset}`}
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            label={<Typography className={classes.label}>EMAIL ADDRESS</Typography>}
```

Comment:
Please refer to comment above about CAPITALCASING

Reply:
Applied